src/i18next.js

```diff
@@ -578,7 +578,7 @@ class I18n extends EventEmitter {
       : 'ltr';
   }
 
-  static createInstance = (options = {}, callback) => new I18n(options, callback)
+  static createInstance(options = {}, callback) { return new I18n(options, callback) }
```

Comment:
we may ask @perrin4869 => https://github.com/i18next/i18next/commit/e401175e1912a837e883923121e997df81ba4111

Reply:
Looks equivalent to me!
I generally prefer to use arrow functions because it saves the need to use curly braces, so it was probably only an esthetics decision, feel free to change it 🙂

Reply:
@perrin4869 to give you a bit of context... in the latest I18next v23 we don't transpile for older browser/platforms... but the notation with `static xxx = function` might bring some issues (does not pass eschecks before es2022). So the idea here is to refactor at the source. See also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static